index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <title>Cookie Stand</title>
+  </head>
+  <body>
+    <link href="style.css" rel="stylesheet" type="text/css"/>
+    <script src="app.js"></script>
```

Comment:
Currently, we're not using app.js in index.html so you don't need to import it. You're actually getting a console error right now in index as a result.